src/test/java/seedu/address/model/person/AddressTest.java

```diff
@@ -31,6 +31,6 @@ public void isValidAddress() {
         // valid addresses
         assertTrue(Address.isValidAddress("Blk 456, Den Road, #01-355"));
         assertTrue(Address.isValidAddress("-")); // one character
-        assertTrue(Address.isValidAddress("Leng Inc; 1234 Market St; San Francisco CA 2349879; USA")); // long address
+        assertTrue(Address.isValidAddress("aaaaabbbbbcccccdddddaaaaabbbbbcccccddddd")); // long address
```

Comment:
"Leng Inc; 1234 Market St; San Francisco CA 2349879; USA" seems like a better address to test than "aaaaabbbbbcccccdddddaaaaabbbbbcccccddddd". The former consist of a mix of alphabets, numbers, special characters and white spaces, whereas the latter is just alphabets. Is there a particular reason why this is changed as such?